src/ConnectorEdgeModule/moduleSettings.example.json

```diff
@@ -0,0 +1,82 @@
+﻿{
+  "RootDirectory": null,
+  "KeyVaultBaseUrl": null,
+  "DisableSubscriptionRestoreService": true,
+  "EnableMessageFilter": false,
+  "ConnectorId": "###",
+  "Logging": {
+    "LogLevel": {
+      "Default": "debug",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+  "OpcUa": {
+    "DefaultServerBrowseDepth": 3,
+    "NodeBrowseDepth": 1,
+    "EnableRegisteredNodes": false,
+    "SubscriptionBatchSize": 100,
+    "ReadBatchSize": 100,
+    "RegisterNodeBatchSize": 100,
+    "AwaitSessionLockTimeoutSecs": 3,
+    "ReconnectIntervalSecs": 10,
+    "NativeSettings": {
```

Comment:
What is the criteria for a setting to be classified as native?
And Native to what?

Reply:
Each setting section has a type (e.g Mqtt or Kafka) and the Native part is to set the config that is native to that type. We do not want to wrap settings of the different communication types. So we made this section where these settings can be given.

Reply:
Makes more sense now.
Once I add the MQTT stuff, it will become a lot more obvious.